tests/common/test_cgroups.py

```diff
@@ -69,6 +67,25 @@ def i_am_root():
     return os.geteuid() == 0
 
 
+def median(lst):
+    data = sorted(lst)
+    l_len = len(data)
+    if l_len < 1:
+        return None
+    if l_len % 2 == 0:
+        return (data[int((l_len - 1) / 2)] + data[int((l_len + 1) / 2)]) / 2.0
+    else:
+        return data[int((l_len - 1) / 2)]
+
+
+def generate_metric_list(lst):
+    return [sum(lst) / len(lst),
+            min(lst),
+            max(lst),
+            median(lst),
+            len(lst)]
+
+
 @skip_if_predicate_false(lambda: False, "TODO: Need unit tests")
```

Comment:
I disabled the old tests since we were rewriting this code... could you check whether we need any of those tests and remove them if not?